### PR TITLE
Adding Hirschmann OS

### PR DESCRIPTION
**Pull request by Darkiros**
Adding Hirschmann OS detection.

Unfortunately I don't have example of website to show because Hirschmann is an industrial equipment and is rarely exposed on the web.

### CODE REVIEW

**Comment by nurbek91** [comment]
Hey mate, yes confirm couldn't find examples

**Comment by Darkiros** [comment]
> Hey mate, yes confirm couldn't find examples

I tested on intern equipment and it works, maybe i can share you a screenshot of the detection ?

**Comment by Darkiros** [comment]
You can find examples at these addresses : 
https://82.141.17.144/
https://82.141.17.146/
https://82.141.17.148/
https://82.141.17.130/
https://82.141.17.158/

**These IP addresses are, accordingly to what is written on them, part of the Hirschmann Remote Demonstration Setup.**

File: src/technologies/h.json [hunk 1]
```diff
@@ -887,6 +887,23 @@
...
+    ],
+    "description": "Hirschmann HiOS is an operating system for industrial network equipment.",
+    "dom": {
+      "h2": {
+        "text": "(HiSecOS-|HiOS-.+-)(\\d+\\.\\d+\\.\\d+)\\;version:\\2"
+      },
+      "div": {
+        "text": "(HiSecOS-|HiOS-.+-)(\\d+\\.\\d+\\.\\d+)\\;version:\\2"
+      }
+    },
```
Comment (nurbek91) [inline src/technologies/h.json:902]: This should help with resources
```
"requires": [
      "Java",
      "Google Web Server"
    ],
```

Comment (nurbek91) [reply src/technologies/h.json:902]: Sorry mate, Google Web Toolkit instead Google Web Server

**Review by nurbek91 — APPROVED** [review]

File: src/technologies/h.json (hunk 1 above)
Comment (Darkiros) [reply src/technologies/h.json:902]: Np, I changed it 👍

**Review by nurbek91 — APPROVED** [review]

**Pull request merged**